Modules/_sqlite/cursor.c

```diff
@@ -867,6 +876,7 @@ _pysqlite_query_execute(pysqlite_Cursor* self, int multiple, PyObject* operation
 
     stmt_reset(self->statement);
     stmt_mark_dirty(self->statement);
+    self->rowcount = self->statement->is_dml ? 0L : -1L;
```

Comment:
`0LL`, `-1LL`.
This makes it easy for readers to understand the type of `rowcount`.